invokeai/app/api/routers/images.py

```diff
@@ -77,7 +78,7 @@ async def upload_image(
         raise HTTPException(status_code=500, detail="Failed to create image")
 
 
-@images_router.delete("/{image_name}", operation_id="delete_image")
+@images_router.delete("/i/{image_name}", operation_id="delete_image")
```

Comment:
Why these routes changes?

Reply:
Concerned that an image name may clobber `images/upload` or `images/batch` (or any other routes we may add in the future). It really shouldn't happen bc the image names are UUIDv4, but this makes it impossible